openadapt/app/tray.py

```diff
@@ -77,6 +76,8 @@ def __init__(self) -> None:
 
         self.app.setQuitOnLastWindowClosed(False)
 
+        crud.release_db_lock()
```

Comment:
It seems odd to release the lock before acquiring it here. What do you think about adding a comment explaining why this is necessary?